ee/cis/macos-26/test/scripts/CIS_2.6.8_fail.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+# CIS 2.6.8 - Ensure an Administrator Password Is Required to Access System-Wide Preferences
+# Flips shared back to true on system.preferences so the query fails.
+plist="/tmp/system.preferences.plist"
+/usr/bin/sudo /usr/bin/security -q authorizationdb read system.preferences > "$plist"
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)